lib/lita/handlers/github_prs/chat_handler.rb

```diff
@@ -51,8 +51,15 @@ def go_live(receiver, repo)
         end
 
         def additional_todos(repo)
-          path = "#{config.extra_templates}/#{repo.short_name}"
-          return File.read(path) if File.exist?(path)
+          classified_repo_name = classify_repo_name(repo.long_name)
+
+          if Module.const_defined?(classified_repo_name)
+            generator = Object.const_get(classified_repo_name).new(repo: repo)
```

Comment:
Since the definition of that class will in most cases be in other repositories I thought it couldnt hurt to be good to be super explicit about what is being sent